charts/newrelic-k8s-metrics-adapter/templates/_helpers.tpl

```diff
@@ -98,13 +13,9 @@ Select a value for the region
 {{- define "newrelic-k8s-metrics-adapter.region" -}}
```

Comment:
From @paologallinaharbur's comments below I see that US is the default, but this might be non-obvious. Could we have a quick comment saying that?